src/core/meshcop/commissioner.cpp

```diff
@@ -312,7 +316,7 @@ otError Commissioner::AddJoiner(const Mac::ExtAddress *aEui64, const char *aPskd
 
         UpdateJoinerExpirationTimer();
 
-        IgnoreError(SendCommissionerSet());
+        SendCommissionerSet();
```

Comment:
If `SendCommissionerSet()` failed here, do we have any mechanism to assure that the Commissioner information is eventually synchronized to Leader?

Reply:
Yes, that would require more changes and beyond the scope of this PR.